geometry_msgs/PoseArray waypoint_list
---
bool result
---
#progress is in percent
uint8 x_progress
uint8 y_progress
uint8 z_progress
uint8 index_progress
